doc/architectural/howto.md

```diff
@@ -16,12 +16,27 @@ to the basic data structures and workflow needed for contributing to
 
 ## Initial setup
 
-Clone the [CBMC repository][cbmc-repo] and build it:
+Clone the [CBMC repository][cbmc-repo] and build it. The build instructions are
+written in a file called COMPILING.md in the top level of the repository. I
+recommend that you build using CMake---this will place all of the CBMC tools in
+a single directory, which you can add to your `$PATH`. For example, if you built
+the codebase with the following two commands at the top level of the repository:
+
+    cmake -DCMAKE_CXX_FLAGS=-fcolor-diagnostics     \
```

Comment:
For non-experts who don't know about CXX_FLAGS: show a basic all-defaults command too

Reply:
I've written an all-defaults command, except that it does still include defines for the path to the compiler. This is needed until there's a fix for #4956. I've also just posted #5339, which updates COMPILING.md to also contain those CMake defines.